From: Luis de Bethencourt <luisbg@kernel.org>
Date: Tue, 16 Jan 2018 15:03:32 +0000
Subject: vxlan: Fix trailing semicolon
Patch-mainline: v4.16-rc1
Git-commit: 5ef7e0ba1039d65c4eefde92056769548b214273
References: bsc#1076830

The trailing semicolon is an empty statement that does no operation.
It is completely stripped out by the compiler. Removing it since it doesn't do
anything.

Fixes: 5f35227ea34b ("net: Generalize ndo_gso_check to ndo_features_check")
Signed-off-by: Luis de Bethencourt <luisbg@kernel.org>
Acked-by: Stephen Hemminger <stephen@networkplumber.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Michal Kubecek <mkubecek@suse.cz>

---
 include/net/vxlan.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/net/vxlan.h b/include/net/vxlan.h
index da7d6b89df77..0ee005e52d4d 100644
--- a/include/net/vxlan.h
+++ b/include/net/vxlan.h
@@ -305,7 +305,7 @@ static inline netdev_features_t vxlan_features_check(struct sk_buff *skb,
 		l4_hdr = ipv6_hdr(skb)->nexthdr;
 		break;
 	default:
-		return features;;
+		return features;
 	}
 
 	if ((l4_hdr == IPPROTO_UDP) &&
-- 
2.16.1

